src/test/scala/edu/berkeley/ce/rockslicing/BlockSpec.scala

```diff
@@ -2,6 +2,8 @@ package edu.berkeley.ce.rockslicing
 
 import org.scalatest._
 import scala.math.sqrt
+import breeze.linalg
```

Comment:
Hmm, why do you have two `breeze` import statements? IntelliJ can help you clean that up automatically.